basiclibrary/src/test/java/basiclibrary/LibraryTest.java

```diff
@@ -0,0 +1,53 @@
+package basiclibrary;/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+import java.util.*;
+
+import basiclibrary.Library;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test public void testSomeLibraryMethod() {
+        Library classUnderTest = new Library();
+        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
+    }
+
+    @Test public void testRollLength(){
+        int[] roller = Library.roll(7);
+        assertTrue(roller.length == 7);
+    }
+
+    @Test public void testRollRange(){
+        int[] roller = Library.roll(7);
+
+        for (int i = 0; i < 7; i++){
+            assertTrue(roller[i] > 0 && roller[i] <= 6);
```

Comment:
Good test 👍, you could even run that loop more times since you're generating random numbers.